mappings/dataprepper/README.md

```diff
@@ -0,0 +1,2 @@
+# ocsf-benthos
```

Comment:
Stale reference needs to be corrected.

Also, I would recommend adding a readme to describe basic details about dataprepper.